client/src/pages/global.less

```diff
@@ -18,20 +18,31 @@ body,
 	color: rgba(0, 0, 0, 0.65) !important;
```

Comment:
I'd never recommend using `!important`, instead find a way to override with class hierarchy or else.